Document extensions required by PTFE (#702)
hashicorp/terraform-website: content/source/docs/enterprise/private/preflight-installer.html.md

following must be present on it:

> * PostgreSQL version 9.4 or greater

Reviewer: I might rather we stick with this wording, and only add the sub-bullet with the slightly stronger wording suggested

Author: But it's not really true given the other statements. I don't want to contradict this statement just a few lines later.

* User with the ability to create/modify/read tables and indices on all schemas created

>   * If the user can not create extensions (ie is not a superuser), then the extensions listed below must be created first

Reviewer: ```suggestion
  * If it's not feasible to have a user with "CREATE EXTENSION", then create the [extensions](#extensions) below before installation
```
